kotlin/samples/todo-android/common/src/main/java/com/squareup/sample/todo/TodoEditorWorkflow.kt

```diff
@@ -97,15 +97,13 @@ class TodoEditorWorkflow : StatelessWorkflow<TodoList, TodoEditorOutput, TodoRen
     props: TodoList,
     context: RenderContext<Nothing, TodoEditorOutput>
   ): TodoRendering {
-    val rawSink = context.makeActionSink<TodoAction>()
-
     // Make event handling idempotent until https://github.com/square/workflow/issues/541 is fixed.
     var eventFired = false
     val sink = object : Sink<TodoAction> {
       override fun send(value: TodoAction) {
         if (eventFired) return
         eventFired = true
-        rawSink.send(value)
+        context.send(value)
```

Comment:
I'm increasingly filled with regret that we didn't go with `RenderContext.render(…)` as the method signature.

Reply:
Just thinking out loud, don't want to distract us from this release, but I bet we could retrofit that pretty easily. Same trick we're doing with `WorkflowAction` — introduce the new method with a default impl that calls the old method, etc.

Reply:
For sure. See #836.